app/workers/commit_monitor_handlers/commit_range/github_pr_commenter/commit_metadata_checker.rb

```diff
@@ -22,6 +22,7 @@ def process_commits(new_commits)
 
       new_commits.each do |commit_sha, data|
         check_for_usernames_in(commit_sha, data["message"])
+        check_for_merge_commit(commit_sha, data["merge_commit?"])
```

Comment:
NOTE `merge_commit?` is set in the commit's `details_hash` here: https://github.com/ManageIQ/miq_bot/blob/master/lib/git_service/commit.rb#L78